src/command-handlers/teamLogs.ts

```diff
@@ -1,26 +1,17 @@
-import { StartAuditLogsQueryParams, startAuditLogsQuery, getAuditLogQueryResults } from '../endpoints/index.js';
+import { getAuditLogs } from '../endpoints/index.js';
 import { getTeamDeviceCredentials, jsonToCsv, epochTimestampToIso } from '../utils/index.js';
-import { GenericLog } from '../types/logs.js';
 import { logger } from '../logger.js';
 
-export const runTeamLogs = async (options: {
-    start: string;
-    end: string;
-    type: string;
```

Comment:
You likely want to update the documentation too as it is explained how to filter by type and category

Reply:
Done